core/server/web/api/canary/admin/routes.js

```diff
@@ -97,6 +97,8 @@ module.exports = function apiRoutes() {
         http(apiCanary.members.importCSV)
     );
 
+    router.get('/members/stripe-connect', shared.middlewares.labs.members, mw.authAdminApi, http(apiCanary.membersStripeConnect.auth));
```

Comment:
Not sure about the naming for `stripe-connect`. We don't have the `-` convention for any other endpoints and usually use `_` as a separator, e.g.: `/email_preview/posts/:id`, `/integrations/:id/api_key/:keyid/refresh`, `/members/:id/signin_urls`

Reply:
Cool - will raise w/ Hannah/Zimo RE naming :+1:

Reply:
Updated to use underscore, I think bothering them with naming q like that is waste of time!